tools/validate_configs/test_configs/vm.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: simple-vm
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: simple-vm
+  region: us-central1
+  zone: us-central1-c
+
+# Documentation for each of the modules used below can be found at
+# https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/README.md
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network1
+    source: modules/network/pre-existing-vpc
+
+  - source: ./modules/compute/vm-instance
```

Comment:
the arbitrary use or non-use of `./` throughout the test_configs directory (A) seems bad to be arbitrary (B) seems bad to begin with `./` instead of testing the binary's copy.

@mr0re1 ?

Reply:
I interpreted the other comment as saying that you were removing this file entirely. Since the whole directory is a mish-mash of this choice, I'm happy to leave as-is, but I think we should address what might be inconsistent approach to testing.